.github/workflows/e2e-master.yaml

```diff
@@ -0,0 +1,44 @@
+name: End to End Tests - master
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Create Kind Cluster
+      uses: helm/kind-action@v1.1.0
+      with:
+        cluster_name: kubernetes-python-e2e-master-${{ matrix.python-version }}
+        # The kind version to be used to spin the cluster up
+        # this needs to be updated whenever a new Kind version is released
+        version: v0.11.1
+        # Update the config here whenever a new client snapshot is performed
+        # This would eventually point to cluster with the latest Kubernetes version
+        # as we sync with Kubernetes upstream
+        config: .github/workflows/kind-configs/cluster-1.18.yaml
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2.1.3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -r requirements.txt
+        python -m pip install -r test-requirements.txt
+    - name: Install package
+      run: python -m pip install -e .
+    - name: Run End to End tests
+      run: pytest -vvv -s kubernetes/e2e_test
```

Comment:
is it possible to pull the main repo and run the e2e tests in the python-base repo using Github Actions?